app/frontend/private/pos/orders/script.js

```diff
@@ -14,18 +14,18 @@ fetch("/api/orders")
     })
     .then((body) => {
         for (let order of body) {
-            const tr = document.createElement("tr");
+            const squareButton = document.createElement("button");
+            squareButton.classList.add("order-button");
```

Comment:
```suggestion
            const squareButton = document.createElement("a");
            squareButton.href = `pos/payment?id=${order.id}`
```

Reply:
I tried this but it goes to http://localhost:3000/pos/orders/pos/payment?id=3

Reply:
oops, should be `/pos/payment?id${order.id}`